engine/lib/river.php

```diff
@@ -808,9 +815,9 @@ function _elgg_river_enable($event, $type, $entity) {
 	LEFT JOIN {$dbprefix}entities AS te ON te.guid = rv.target_guid
 	SET rv.enabled = 'yes'
 	WHERE (
-			(se.enabled = 'yes' OR se.guid IS NULL) AND 
+			(se.enabled = 'yes' OR se.guid IS NULL) AND
```

Comment:
That's fine, I don't think the spaces are necessary here, the line break should count as a space as it's not appending it as separate variables, it's all part of the same multi-line variable definition.  If it didn't break the SQL when you used it then we're good.